src/libstore/sandbox-defaults.sb

```diff
@@ -85,3 +91,7 @@
        (literal "/etc")
        (literal "/var")
        (literal "/private/var/tmp"))
+
+; This is used by /bin/sh on macOS 10.15 and later.
+(allow file*
+       (literal "/private/var/select/sh"))
```

Comment:
We actually want to explicitly *disallow* access to this, because this file is what’s used to control whether `/bin/sh` is `bash` or `zsh`. If we disallow access it will always resolve to bash. If we allow access then the user can configure it to be `zsh` instead.

Reply:
Thought I tested that and it failed, but looks like it's indeed only an error message.

```
$ sandbox-exec -f sandbox.sb /bin/sh -c "echo ok"
Error opening /private/var/select/sh: Operation not permitted
ok
```

Luckily it doesn't really matter and not having to spew out errors out by default is nice.
```
Failed to exec /bin/zsh as variant for /bin/sh (1: Operation not permitted). Falling back to /bin/bash.
ok
```